docs: refactor README, CONTRIBUTING (#179)
openebs/community: README.md

# OpenEBS Community

[![CNCF Status](https://img.shields.io/badge/cncf%20status-sandbox-blue.svg)](https://www.cncf.io/projects/openebs/)
[![LICENSE](https://img.shields.io/github/license/openebs/openebs.svg)](./LICENSE)
[![FOSSA Status](https://app.fossa.com/api/projects/custom%2B162%2Fgithub.com%2Fopenebs%2Fopenebs.svg?type=shield&issueType=license)](https://app.fossa.com/projects/custom%2B162%2Fgithub.com%2Fopenebs%2Fopenebs?ref=badge_shield&issueType=license)
[![CLOMonitor](https://img.shields.io/endpoint?url=https://clomonitor.io/api/projects/cncf/openebs/badge)](https://clomonitor.io/projects/cncf/openebs)
[![Slack](https://img.shields.io/badge/chat-slack-ff1493.svg?style=flat-square)](https://kubernetes.slack.com/messages/openebs)
[![Community Meetings](https://img.shields.io/badge/Community-Meetings-blue)](https://us05web.zoom.us/j/87535654586?pwd=CigbXigJPn38USc6Vuzt7qSVFoO79X.1)
[![Artifact HUB](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/openebs)](https://artifacthub.io/packages/helm/openebs/openebs)

> [!Important]

> > OpenEBS is an umbrella :open_umbrella: project, composed as a federation of individual sub-projects. The project files in this repository apply to the umbrella project, every sub-project in the OpenEBS organization.

Reviewer: umbrella project and every

[![CLOMonitor](https://img.shields.io/endpoint?url=https://clomonitor.io/api/projects/cncf/openebs/badge)](https://clomonitor.io/projects/cncf/openebs)
[![Slack](https://img.shields.io/badge/chat-slack-ff1493.svg?style=flat-square)](https://kubernetes.slack.com/messages/openebs)
[![Community Meetings](https://img.shields.io/badge/Community-Meetings-blue)](https://us05web.zoom.us/j/87535654586?pwd=CigbXigJPn38USc6Vuzt7qSVFoO79X.1)
[![Artifact HUB](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/openebs)](https://artifacthub.io/packages/helm/openebs/openebs)

[![OpenEBS Welcome to Community](images/coffe-shop-hacker-pixel-art_HERO_banner.png)](https://www.openebs.io/)

> [!Important]
> OpenEBS is an umbrella :open_umbrella: project, composed as a federation of individual sub-projects. The project files in this repository apply to the umbrella project and every sub-project in the OpenEBS organization.

## What is OpenEBS?

> OpenEBS is an open-source Container Native Storage solution that provides persistent storage for Kubernetes workloads. It enables dynamic provisioning of storage resources using containerized storage controllers, making it highly flexible and cloud-native. OpenEBS supports various storage engines, including LocalPVs for direct node storage and Replicated PV advanced data replication and resilience. It is desiganed to integrate seamlessly with Kubernetes, offering benefits like storage policies, reize, thin-provisiong, snapshots, and restore capabilities, making it an ideal choice for stateful applications. To read more about OpenEBS, check [here](https://github.com/openebs/openebs/).

Reviewer: You could copy this from the other repos. The same typos are present here as well.

> ## Who we are?

Reviewer: Who are we?
Fix in other repos if applicable.

Reviewer: Alternative
Remove the question-mark and keep as "Who we are" unchanged.

We are a community of innovative Data Storage engineers that contribute to this project. OpenEBS is the result of hundreds of amazing individuals, contributors and storage engineers who provide brainstorming ideas, feedback, code reviews and high-quality code to the project. Anyone who is passionate about storage and data is welcome here. <BR>

## Community

This Community repository is the ```central location``` for project info and files like our governance, contribution, code of conduct, security and vision for the OpenEBS organization laid out by the Maintainers. You'll also find a [Top level Issues List](https://github.com/openebs/openebs/issues) for Organization related issues that span across products/projects. We also provide a [main Discussions forum](https://github.com/openebs/openebs/discussions) where **```everyone```** can collaborate and discuss anything.

- Homepage: [openebs.io](https://openebs.io/)
- Mailing list: openebs-team@googlegroups.com

> - Slack: [openebs slack](https://kubernetes.slack.com/messages/openebs)

Reviewer 2: nit: just because below on twitter we have it as simply openebs 

```suggestion
- Slack: [openebs](https://kubernetes.slack.com/messages/openebs)
```

and operational matters. We curate the operations of the project, product, roadmaps, initiatives, all engineering/code activities and all events (including conferences).

 [<img alt="Container Storage Interface group" src="images/CNCF_csi-horizontal-color_2024.png" width="320">](https://github.com/kubernetes/community/tree/master/sig-storage)  [<img alt="Storage Technical Advisory Group" src="images/CNCF_tag-storage-horizontal-color_2024.png" width="320">](https://github.com/cncf/tag-storage)  &emsp; &emsp; [<img alt="Cloud Native Community Groups" src="images/CNCF_cncg-icon-color_2024.png" width="200">](https://github.com/cncf/communitygroups)

## How can I get involved?

One of the easiest ways to contribute is to participate in discussions at community engagements.

If you're looking for something to work on, read the [contribution guidelines](./CONTRIBUTING.md) and then you start by looking for GitHub issues, marked with `kind/good-first` or the `kind/help-wanted` labels.

And, we can always use more testing, have more and improved docs, or just write a blog post on what you have discovered whilst using OpenEBS.

## Code of Conduct

> OpenEBS adopts [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).

Reviewer 2: ```suggestion
OpenEBS adopts the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/HEAD/code-of-conduct.md).
```

> This open governance applies to all repos under `openebs` org. Enjoy coding and collaboration in OSS world!

Reviewer: Hope it is OSS and not 0SS?

Reviewer: I validated, it is OSS. Ignore.

> This open governance applies to all repos under `openebs` org. Enjoy coding and collaboration in OSS world!

Reviewer 2: ```suggestion
This open governance applies to all repos under the `openebs` org. Enjoy coding and collaboration in OSS world!
```

[![LICENSE](https://img.shields.io/github/license/openebs/openebs.svg)](./LICENSE)
[![FOSSA Status](https://app.fossa.com/api/projects/custom%2B162%2Fgithub.com%2Fopenebs%2Fopenebs.svg?type=shield&issueType=license)](https://app.fossa.com/projects/custom%2B162%2Fgithub.com%2Fopenebs%2Fopenebs?ref=badge_shield&issueType=license)
[![CLOMonitor](https://img.shields.io/endpoint?url=https://clomonitor.io/api/projects/cncf/openebs/badge)](https://clomonitor.io/projects/cncf/openebs)
[![Slack](https://img.shields.io/badge/chat-slack-ff1493.svg?style=flat-square)](https://kubernetes.slack.com/messages/openebs)
[![Community Meetings](https://img.shields.io/badge/Community-Meetings-blue)](https://us05web.zoom.us/j/87535654586?pwd=CigbXigJPn38USc6Vuzt7qSVFoO79X.1)
[![Artifact HUB](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/openebs)](https://artifacthub.io/packages/helm/openebs/openebs)

> [!Important]
> OpenEBS is an umbrella :open_umbrella: project, composed as a federation of individual sub-projects. The project files in this repository apply to the umbrella project and every sub-project in the OpenEBS organization.

> ## What is OpenEBS?

Reviewer 2: ```suggestion
## What is OpenEBS
```

- [Local PV Hostpath](https://github.com/openebs/dynamic-localpv-provisioner/)
- [Local PV ZFS](https://github.com/openebs/zfs-localpv)
- [Local PV LVM](https://github.com/openebs/lvm-localpv)
- [Mayastor](https://github.com/openebs/mayastor)

## Who we are

We are a community of innovative Data Storage engineers that contribute to this project. OpenEBS is the result of hundreds of amazing individuals, contributors and storage engineers who provide brainstorming ideas, feedback, code reviews and high-quality code to the project. Anyone who is passionate about storage and data is welcome here. <BR>

## Community

This Community repository is the ```central location``` for project info and files like our governance, contribution, code of conduct, security and vision for the OpenEBS organization laid out by the Maintainers. You'll also find a [Top level Issues List](https://github.com/openebs/openebs/issues) for Organization related issues that span across products/projects. We also provide a [main Discussions forum](https://github.com/openebs/openebs/discussions) where **```everyone```** can collaborate and discuss anything.

- Homepage: [openebs.io](https://openebs.io/)

> - Mailing list: openebs-team@googlegroups.com

Reviewer 3: Mailing list?

Author: What's the issue?

Reviewer 3: A mailing list is like a address book/email group where you register and then the folks who own the group send you emails monthly. E.g.: https://lkml.org/

Ours is a `Maintainers' email`.

Author: No this not maintainer's email, there where users are supposed to mail to for queries and stuffs.
Check this https://github.com/cubefs/cubefs?tab=readme-ov-file#community

## Documents

- [Official Documentation](https://openebs.io/docs)
- [Governance](./GOVERNANCE.md)
- [Contributing Guidelines](./CONTRIBUTING.md)
- [Security Guidelines](./SECURITY.md)
- [Vision](./VISION.md)
- [Roadmap Tracker](https://github.com/orgs/openebs/projects/78)

## Project Leadership team

This Community is managed by the OpenEBS [Maintainers](./MAINTAINERS.md) within the OpenEBS project team. We liaise with the Linux Foundation and CNCF on project, governance
and operational matters. We curate the operations of the project, product, roadmaps, initiatives, all engineering/code activities and all events (including conferences).

 [<img alt="Container Storage Interface group" src="images/CNCF_csi-horizontal-color_2024.png" width="320">](https://github.com/kubernetes/community/tree/HEAD/sig-storage)  [<img alt="Storage Technical Advisory Group" src="images/CNCF_tag-storage-horizontal-color_2024.png" width="320">](https://github.com/cncf/tag-storage)  &emsp; &emsp; [<img alt="Cloud Native Community Groups" src="images/CNCF_cncg-icon-color_2024.png" width="200">](https://github.com/cncf/communitygroups)

> ## How can I get involved?

Reviewer 2: ```suggestion
## How to get involved
```

One of the easiest ways to contribute is to participate in discussions at community engagements.

> If you're looking for something to work on, read the [contribution guidelines](./CONTRIBUTING.md) and then you start by looking for GitHub issues, marked with `kind/good-first` or the `kind/help-wanted` labels.

Reviewer 3: I think these two labels are named 'good first issue' and 'help wanted' respectively.

Author: Let's leave out the label discussion out of this PR. I will raise a different one once we all decide.
